<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ICWS-1157" name="Default test" tests="8" failures="0" timestamp="21 Mar 2017 10:46:05 GMT" time="0.16" errors="0">
  <testcase name="@BeforeMethod setUp" time="0.0" classname="com.cfs.index.ServiceRequestIndex">
    <failure type="java.net.MalformedURLException" message="For input string: &amp;quot;null&amp;quot;">
      <![CDATA[java.net.MalformedURLException: For input string: "null"
	at java.net.URL.<init>(URL.java:627)
	at java.net.URL.<init>(URL.java:490)
	at java.net.URL.<init>(URL.java:439)
	at com.cfs.init.SeleniumInit.setUp(SeleniumInit.java:117)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:585)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
Caused by: java.lang.NumberFormatException: For input string: "null"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Integer.parseInt(Integer.java:580)
	at java.lang.Integer.parseInt(Integer.java:615)
	at java.net.URLStreamHandler.parseURL(URLStreamHandler.java:216)
	at java.net.URL.<init>(URL.java:622)
	... 29 more
]]>
    </failure>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="requestByBOLFlow" time="0.001" classname="com.cfs.index.ServiceRequestIndex">
    <skipped/>
  </testcase> <!-- requestByBOLFlow -->
  <testcase name="planPostpaidRequest" time="0.0" classname="com.cfs.index.ServiceRequestIndex">
    <skipped/>
  </testcase> <!-- planPostpaidRequest -->
  <testcase name="approvePostpaidRequest" time="0.0" classname="com.cfs.index.ServiceRequestIndex">
    <skipped/>
  </testcase> <!-- approvePostpaidRequest -->
  <testcase name="viewCreatedJob" time="0.0" classname="com.cfs.index.JobIndex">
    <skipped/>
  </testcase> <!-- viewCreatedJob -->
  <testcase name="planCreatedJob" time="0.0" classname="com.cfs.index.JobIndex">
    <skipped/>
  </testcase> <!-- planCreatedJob -->
  <testcase name="confirmContainerArrival" time="0.0" classname="com.cfs.index.JobIndex">
    <skipped/>
  </testcase> <!-- confirmContainerArrival -->
  <testcase name="completeDevanning" time="0.0" classname="com.cfs.index.JobIndex">
    <skipped/>
  </testcase> <!-- completeDevanning -->
  <testcase name="completedRequest" time="0.0" classname="com.cfs.index.ServiceRequestIndex">
    <skipped/>
  </testcase> <!-- completedRequest -->
</testsuite> <!-- Default test -->
